src/managers/retention-policies.js

```diff
@@ -322,4 +324,56 @@ RetentionPolicies.prototype.getAllFileVersionRetentions = function(options, call
 	return this.client.wrapWithDefaultHandler(this.client.get)(apiPath, params, callback);
 };
 
+/**
+ * Get files under retention by each assignment
+ * To use this feature, you must have the manage retention policies scope enabled
+ * for your API key via your application management console.
+ *
+ * API Endpoint: '/retention_policy_assignments/:assignmentID/files_under_retention'
+ * Method: GET
+ *
+ * @param {string} assignmentID - The Box ID of the policy assignment object to fetch
+ * @param {Object} [options] - Additional options for the request. Can be left null in most cases.
+ * @param {string} [options.fields] - Comma-separated list of fields to include in the response: disposition_at, winning_retention_policy
+ * @param {int} [options.limit] - The maximum number of items to return in a page
+ * @param {string} [options.marker] - Paging marker, left blank to begin paging from the beginning
+ * @param {Function} [callback] - Pass the file version retention record if successful, error otherwise
+ * @returns {Promise<Object>} A promise resolving to the collection of retention records
+ */
+RetentionPolicies.prototype.getFilesUnderRetentionForAssignment = function(assignmentID, options, callback) {
+
+	var apiPath = urlPath(ASSIGNMENTS_PATH, assignmentID, FILES_UNDER_RETENTION_SUBRESOURCE),
+		params = {
+			qs: options
+		};
+
+	return this.client.wrapWithDefaultHandler(this.client.get)(apiPath, params, callback);
+};
+
+/**
+ * Get file versions under retention by each assignment
+ * To use this feature, you must have the manage retention policies scope enabled
+ * for your API key via your application management console.
+ *
+ * API Endpoint: '/retention_policy_assignments/:assignmentID/files_under_retention'
```

Comment:
minor: `.../files_versions_under_retention` instead of `.../files_under_retention`